Tests/FluentAssertions.Equivalency.Specs/CollectionSpecs.cs

```diff
@@ -556,6 +556,320 @@ public void When_a_deeply_nested_property_of_a_collection_with_an_invalid_value_
+                            },
+                            new
+                            {
+                                Number = 2,
+                                Text = "Actual"
+                            }
+                        }
+                    }
+                };
+
+                var expected = new
+                {
+                    Text = "Expected",
+                    Level = new
+                    {
+                        Collection = new[]
+                        {
+                            new
+                            {
+                                Number = 1,
+                                Text = "Text"
+                            },
+                            new
+                            {
+                                Number = 2,
+                                Text = "Expected"
+                            }
+                        }
+                    }
+                };
+
+                // Act / Assert
+                subject.Should().BeEquivalentTo(expected,
+                    options => options
+                        .For(x => x.Level.Collection).Exclude(x => x.Text)
+                        .Excluding(x => x.Text));
+            }
+
+            [Fact]
+            public void A_nested_exclusion_can_be_preceded_by_a_root_level_exclusion()
+            {
+                // Arrange
+                var subject = new
+                {
+                    Text = "Actual",
+                    Level = new
+                    {
+                        Collection = new[]
+                        {
+                            new
+                            {
+                                Number = 1,
+                                Text = "Text"
+                            },
+                            new
+                            {
+                                Number = 2,
+                                Text = "Actual"
+                            }
+                        }
+                    }
+                };
+
+                var expected = new
+                {
+                    Text = "Expected",
+                    Level = new
+                    {
+                        Collection = new[]
+                        {
+                            new
+                            {
+                                Number = 1,
+                                Text = "Text"
+                            },
+                            new
+                            {
+                                Number = 2,
+                                Text = "Expected"
+                            }
+                        }
+                    }
+                };
+
+                // Act / Assert
+                subject.Should().BeEquivalentTo(expected,
+                    options => options
+                        .Excluding(x => x.Text)
+                        .For(x => x.Level.Collection).Exclude(x => x.Text));
```

Comment:
🤔 How about the scenario why you have two sets of `For` and `Exclude`? Is that possible?

Reply:
I added a test for that 👍